frontend/src/metabase/admin/embedding/components/ThemeListing/EmbeddingThemeListingApp.tsx

```diff
@@ -24,14 +25,13 @@ export function EmbeddingThemeListingApp() {
   const { data: themes, isLoading } = useListEmbeddingThemesQuery();
   const [createTheme] = useCreateEmbeddingThemeMutation();
   const [sendToast] = useToast();
+  const defaultTheme = useDefaultEmbeddingTheme();
 
   const handleCreateTheme = async () => {
     try {
       await createTheme({
         name: t`Untitled theme`,
-
-        // TODO(EMB-962): Add default theme colors based on instance's appearance settings
-        settings: {},
+        settings: defaultTheme,
```

Comment:
It feels a little odd to me that this is called `defaultTheme` but it's assigned to the `settings` propery of what's passed to `createTheme`. Maybe a more fitting name would be `defaultThemeSettings`, returned by `useDefaultEmbeddinThemeSettings` (although this is a bit long)?

Reply:
Fixed in https://github.com/metabase/metabase/pull/65499/commits/ff3e879376295fceac51ecc63a98c0487110ee2a, thanks! This is mostly my fault as I usually name the theme object `MetabaseTheme`, and that was all well and good until we have the stored themes on the backend, so I had to call the field something like `settings` to avoid `theme.theme` 😆